sites/docs/pages/components/date-input/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Date Input
+sidebar_position: 1
+queries: 
+- orders_by_day.sql
+---
+
+Creates a date picker that can be used to filter a query.
+
+To see how to filter a query using an input component, see [Filters](/core-concepts/filters).
+
+```sql filtered_query
+select 
+*
+from ${orders_by_day}
+where day = '${inputs.date_filtering_a_query.value}'
+```
+
+<DocTab>
+    <div slot='preview'>
+            <DateInput
+                name=date_filtering_a_query
+                data={orders_by_day}
+                dates=day
+            />
+
+            <BarChart
+                data={filtered_query}
+                x=day
+                y=sales
+                y2=num_orders
+            />
+    </div>
+
+````markdown
+
+```sql filtered_query
+select 
+    *
+from ${orders_by_day}
+where day = '${inputs.range_filtering_a_query.value}'
```

Comment:
This example chart looks odd, A barchart with a single day is rarely used.

I think you'd be using this as a one sided filter. 

```suggestion
where day > '${inputs.range_filtering_a_query.value}'
```

Reply:
oh nice yeah, i like this better